engine/ref_api.h

```diff
@@ -103,6 +104,15 @@ typedef enum
 	DEMO_QUAKE1
 } demo_mode;
 
+enum
+{
+	REF_WINDOW_TYPE_WIN32 = 1, // HWND
```

Comment:
There is no point to add numbers here. :)

Also it would be nice if you gave this enum a type and used it instead of abstract `int`.